src/NzbDrone.Core/IndexerSearch/NzbSearchService.cs

```diff
@@ -216,13 +221,21 @@ private List<DownloadDecision> SearchSpecial(Series series, List<Episode> episod
             return Dispatch(indexer => indexer.Fetch(searchSpec), searchSpec);
         }
 
-        private List<DownloadDecision> SearchAnimeSeason(Series series, List<Episode> episodes)
+        private List<DownloadDecision> SearchAnimeSeason(Series series, List<Episode> episodes, int seasonNumber)
         {
-            var downloadDecisions = new List<DownloadDecision>();
-
-            foreach (var episode in episodes)
-            {
-                downloadDecisions.AddRange(SearchAnime(series, episode));
+            var downloadDecisions = new List<DownloadDecision>();
+
+            // Season 0 indicates specials, not regular episodes
+            if (seasonNumber == 0)
+            {
+                downloadDecisions.AddRange(SearchSpecial(series, episodes));

```

Comment:
What about specials that have an absolute episode number, this should search those episodes normally and sure special search if they are missing.